### ISSUE TITLE
no-pending-tests doesn't catch skipped tests

### ISSUE THREAD

**Issue by JavaScriptBach** [lo1tuma/eslint-plugin-mocha#373]
The following are not caught by `mocha/no-pending-tests` in eslint-plugin-mocha 10.5.0:

```javascript
it.skip("foo", function () {});
it.skip("bar", () => "hello world");
```

This looks like a regression; they were caught by `no-skipped-tests` in 10.4.3.

**Comment by JavaScriptBach** [comment]
Oh nevermind; I see what the problem is. `main` contains a lot of changes that haven't been released yet, and I was reading those docs.
